mtags/src/main/scala-2/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -199,7 +199,10 @@ trait Completions { this: MetalsGlobal =>
               if (byFuzzy != 0) byFuzzy
               else {
                 val byIdentifier =
-                  IdentifierComparator.compare(o1.sym.name, o2.sym.name)
+                  IdentifierComparator.compare(
+                    o1.sym.name.decode,
+                    o2.sym.name.decode
+                  )
```

Comment:
Some changes in test data for Scala2 were triggered by this change.
For example `::` was compared as `$colon$colon` that resulted to the difference in ordering between 2 and 3